### PR TITLE
AT_09.06.001 | My views | Delete View

### PR DESCRIPTION
**Pull request by Iriskus**
https://trello.com/c/mlCPlW6n/884-at0906001-my-views-delete-view

### CODE REVIEW

File: cypress/e2e/myViewDeleteView.cy.js [hunk 1]
```diff
@@ -0,0 +1,24 @@
+/// <reference types="cypress"/>
+
+
+describe('myViewsDeleteView', () => {
+    it('TC_09.06.001 | <My views> Delete View', function () {
```
Comment (Rom-T) [inline cypress/e2e/myViewDeleteView.cy.js:5]: Rename please it to `'AT_09.......'`

File: cypress/e2e/myViewDeleteView.cy.js [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+        cy.get('#tasks > div:nth-child(5) > span > a > span.task-link-text').realHover();
+        cy.get('#tasks > div:nth-child(5) > span > a > span.task-link-text').click({ force: true });
```
Comment (Rom-T) [inline cypress/e2e/myViewDeleteView.cy.js:7]: Actions for 1 element can be listed through a dot
cy.get('#tasks > div:nth-child(5) > span > a > span.task-link-text').realHover().click({ force: true });

File: cypress/e2e/myViewDeleteView.cy.js [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+        cy.get('a[href$="/newJob"]').click();
+        cy.get('#name').type('First Job');
```
Comment (Rom-T) [inline cypress/e2e/myViewDeleteView.cy.js:9]: please move `'First Job'` to fixtures

File: cypress/e2e/myViewDeleteView.cy.js [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+        cy.get('.hudson_model_FreeStyleProject').click();
+        cy.get('#ok-button').click();
+        cy.get('.jenkins-button--primary').click();
+        cy.get(':nth-child(1) > .model-link').click();
+        cy.get('.addTab').click();
+        cy.get('#name').type('MyView');
```
Comment (Rom-T) [inline cypress/e2e/myViewDeleteView.cy.js:15]: please move `'MyView'` to fixtures

File: cypress/e2e/myViewDeleteView.cy.js [hunk 5]
```diff
@@ -0,0 +1,24 @@
...
+        cy.get('label[for="hudson.model.MyView"]').click();
+        cy.get('#ok').click();
+        cy.get('.active > a').click();
+        cy.get('.task-link-text').contains('Delete View').click({ force: true });
```
Comment (Rom-T) [inline cypress/e2e/myViewDeleteView.cy.js:19]: please move `'Delete View'` to fixtures

File: cypress/e2e/myViewDeleteView.cy.js [hunk 6]
```diff
@@ -0,0 +1,24 @@
...
+        cy.get('#main-panel > form > button').click();
+        cy.get('.addTab').contains('MyView').should('not.exist');
```
Comment (Rom-T) [inline cypress/e2e/myViewDeleteView.cy.js:21]: please move `'MyView'` to fixtures

File: cypress/e2e/myViewDeleteView.cy.js [hunk 7]
```diff
@@ -1,24 +1,26 @@
 /// <reference types="cypress"/>
 
+const myViewName = 'First Job'
+const myView = 'MyView'
+const deleteView = 'Delete View'
```
Comment (Rom-T) [inline cypress/e2e/myViewDeleteView.cy.js:5]: Please move `data` in the fixtures

Comment (Iriskus) [reply cypress/e2e/myViewDeleteView.cy.js:5]: done

File: cypress/e2e/myViewDeleteView.cy.js (hunk 6 above)
Comment (Iriskus) [reply cypress/e2e/myViewDeleteView.cy.js]: done

File: cypress/e2e/myViewDeleteView.cy.js (hunk 5 above)
Comment (Iriskus) [reply cypress/e2e/myViewDeleteView.cy.js]: done

File: cypress/e2e/myViewDeleteView.cy.js (hunk 4 above)
Comment (Iriskus) [reply cypress/e2e/myViewDeleteView.cy.js]: done

File: cypress/e2e/myViewDeleteView.cy.js (hunk 3 above)
Comment (Iriskus) [reply cypress/e2e/myViewDeleteView.cy.js]: done

File: cypress/e2e/myViewDeleteView.cy.js (hunk 2 above)
Comment (Iriskus) [reply cypress/e2e/myViewDeleteView.cy.js]: done

File: cypress/e2e/myViewDeleteView.cy.js (hunk 1 above)
Comment (Iriskus) [reply cypress/e2e/myViewDeleteView.cy.js]: done

**Comment by Iriskus** [comment]
@Rom-T

**Comment by Iriskus** [comment]
@Rom-T

File: cypress/e2e/myViewDeleteView.cy.js (hunk 7 above)
Comment (Rom-T) [reply cypress/e2e/myViewDeleteView.cy.js:5]: No, you must transfer the data to json fixtures and do an import

**Review by Rom-T — APPROVED** [review]

**Review by Alina-Fedarenka — APPROVED** [review]

**Pull request merged**